readme_renderer/integration/distutils.py

```diff
@@ -26,6 +28,16 @@ def check_restructuredtext(self):
         Checks if the long string fields are reST-compliant.
         """
         data = self.distribution.get_long_description()
+        content_type = getattr(
+            self.distribution.metadata, 'long_description_content_type', None)
```

Comment:
I'm not sure when `self.distribution.metadata` came to be -- is this available in all version of distutils? Otherwise we might want to test whether these attributes exist.

Reply:
It's present in Python 2.7 [here](https://github.com/python/cpython/blob/77f0a41d72886513f5e9277e5587455745c29ac1/Lib/distutils/dist.py#L151), and going back quite a bit it looks like it's been around for at least [18 years](https://github.com/python/cpython/blame/39d59b47b70d557e0aec5c8f35ede0a7d73f4edc/Lib/distutils/dist.py#L121).

Reply:
That's probably far enough 😉